### DISCUSSION TITLE
Upload Alpine Image to Dockerhub

### DISCUSSION THREAD

**Discussion by LaurinStreng** [ltb-project/self-service-password#1011] [General]
Hey @coudot,

Big thanks for providing the Alpine Image in 1.7.0.

Would it be possible to upload that Image to DockerHub with an alpine tag?

That would be amazing.

**Comment by coudot** [comment]
Hello,

alpine images are available on docker hub: https://hub.docker.com/r/ltbproject/self-service-password/tags
